packages/docusaurus-theme-openapi/src/theme/ApiDemoPanel/VSCode/index.tsx

```diff
@@ -8,7 +8,7 @@
 import React, { useState } from "react";
 
 import Editor, { Monaco } from "@monaco-editor/react";
-import useThemeContext from "@theme/hooks/useThemeContext";
+import { useColorMode } from "@docusaurus/theme-common";
```

Comment:
I think this is equivalent, but needs to be checked

Reply:
I believe this is correct